src/client/viewSandbox.js

```diff
@@ -350,15 +353,25 @@ const init = () => {
         } else {
           validateOutput.innerHTML = 'Invalid';
         }
+
+        validateButton.disabled = false;
+        validateButton.textContent = 'Validate';
```

Comment:
You should probably run this inside a `finally` so that if there's an error it gets reset.